src/org/usfirst/frc/team694/robot/commands/LiftUpCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc.team694.robot.commands;
+
+import org.usfirst.frc.team694.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class LiftUpCommand extends Command {
+
+    public LiftUpCommand() {
+        requires(Robot.lift);
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+        Robot.lift.goUp();
```

Comment:
Same problem as LiftDownCommand and LiftStopCommand: Execute never runs when isFinished returns true. Consider using InstantCommand